crates/telio-firewall/src/firewall.rs

```diff
@@ -1118,6 +1170,101 @@ impl Firewall for StatefullFirewall {
             telio_log_error!("TCP cache poisoned");
         };
 
+        if let Ok(mut udp_conn_cache) = self.udp.lock() {
```

Comment:
Can we split `reset_connections()` into two functions for TCP and UDP? This should make the `reset_connections()` be the orchestractor of reset logic with each function doing only it's part, would be easier on the eyes

Reply:
Yeah, that's a good idea :+1:

Reply:
Done